spec/index.bs

```diff
@@ -980,6 +980,34 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         <tr><th>Limit name <th>Type <th>[=limit/Better=] <th>[=limit/Default=]
     </thead>
 
+    <tr><td><dfn>maxTextureDimension1D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>4096
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}}
+        for which the {{GPUTextureDescriptor/dimension}} is {{GPUTextureDimension/"1d"}}
+        when creating a {{GPUTexture}}.
+
+    <tr><td><dfn>maxTextureDimension2D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>4096
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}} and {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/height}}
+        for which the {{GPUTextureDescriptor/dimension}} is {{GPUTextureDimension/"2d"}}
+        when creating a {{GPUTexture}}.
+
+    <tr><td><dfn>maxTextureDimension3D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>256
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}}, {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/height}} and {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/depth}}
+        for which the {{GPUTextureDescriptor/dimension}} is {{GPUTextureDimension/"3d"}}
+        when creating a {{GPUTexture}}.
+
+    <tr><td><dfn>maxTextureArrayLayer</dfn>
```

Comment:
Does this have to be singular? If the "max texture array layer" value is 256, it's confusing, since what it means is the number of array layers, not an array layer. Perhaps, we can rename this to `maxTextureArrayLayers`?

Reply:
My bad. I missed `s`. Done.